File: app/src/main/java/com/example/mitroshin/pagingdemo/model/entity/Photo.java
package com.example.mitroshin.pagingdemo.model.entity;

import com.google.gson.annotations.SerializedName;

public class Photo {

    @SerializedName("id")
    public final String id;

    @SerializedName("created_at")
    public final String createdAt;

    @SerializedName("updated_at")
    public final String updatedAt;

    @SerializedName("width")
    public final int width;

    @SerializedName("height")
    public final int height;

    @SerializedName("color")
    public final String color;

    @SerializedName("likes")
    public final int likes;

    @SerializedName("liked_by_user")
    public final boolean likedByUser;

    @SerializedName("description")
    public final String description;

    @SerializedName("user")
    public final User user;

    @SerializedName("urls")
    public final Urls urls;

    @SerializedName("links")
    public final Links links;

    public Photo(String id,
                 String createdAt,
                 String updatedAt,
                 int width,
                 int height,
                 String color,
                 int likes,
                 boolean likedByUser,
                 String description,
                 User user,
                 Urls urls,
                 Links links) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.width = width;
        this.height = height;
        this.color = color;
        this.likes = likes;
        this.likedByUser = likedByUser;
        this.description = description;
        this.user = user;
        this.urls = urls;
        this.links = links;
    }

    @Override
    public String toString() {
        return "Photo{" +
                "id='" + id + '\'' +
                ", createdAt='" + createdAt + '\'' +
                ", updatedAt='" + updatedAt + '\'' +
                ", width=" + width +
                ", height=" + height +
                ", color='" + color + '\'' +
                ", likes=" + likes +
                ", likedByUser=" + likedByUser +
                ", description='" + description + '\'' +
                ", user=" + user +
                ", urls=" + urls +
                ", links=" + links +
                '}';
    }
}

File: app/src/main/java/com/example/mitroshin/pagingdemo/model/entity/Photo.kt
package com.example.mitroshin.pagingdemo.model.entity

import com.google.gson.annotations.SerializedName

class Photo(
        @field:SerializedName("id")
        val id: String,

        @field:SerializedName("created_at")
        val createdAt: String,

        @field:SerializedName("updated_at")
        val updatedAt: String,

        @field:SerializedName("width")
        val width: Int,

        @field:SerializedName("height")
        val height: Int,

        @field:SerializedName("color")
        val color: String,

        @field:SerializedName("likes")
        val likes: Int,

        @field:SerializedName("liked_by_user")
        val likedByUser: Boolean,

        @field:SerializedName("description")
        val description: String,

        @field:SerializedName("user")
        val user: User,

        @field:SerializedName("urls")
        val urls: Urls,

        @field:SerializedName("links")
        val links: Links)